lib/tests/fixtures/facts/external/posix/execution/error_message

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env sh
+echo error message! >&2
```

Comment:
We also run batch and cmd files on Windows. That's more what I was thinking of, but batch and PS are good.